packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -273,3 +273,4 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
     return cacao
   }
 }
+
```

Comment:
rm extraneous change to this file, helps keep the git history clean when files are only updated by real changes